test/plugins/features/movies/controller.test.js

```diff
@@ -23,4 +24,49 @@ describe('movie controller', () => {
 
   });
 
+  describe('list', () => {
+
+    beforeEach(() => {
+      const movieOne = { title: 'Batman Begins', release_year: 2005 };
+      const movieTwo = { title: 'The Dark Knight', release_year: 2008 };
+      const movieThree = { title: 'The Dark Knight Rises', release_year: 2012 };
+
+      return Knex('movies').insert([movieOne, movieTwo, movieThree]);
+    });
+
+    it('lists all movies', () => {
+      const query = {};
+
+      return Controller.list(query)
+      .then((movies) => {
+        expect(movies).to.have.lengthOf(3);
+      });
+    });
+
+    it('filters by title', () => {
+      const query = { title: 'the dark knight' };
+
+      return Controller.list(query)
+      .then((movies) => {
+        expect(movies).to.have.lengthOf(1);
+
+        movies.forEach((movie) => {
+          expect(movie.get('title')).to.eql('The Dark Knight');
+        });
+      });
+    });
+
+    it('filters by release_year', () => {
+      const query = { release_year: 2012 };
+
+      return Controller.list(query)
+      .then((movies) => {
+        expect(movies).to.have.lengthOf(1);
+
+        movies.forEach((movie) => {
```

Comment:
Since the result only has a lengthOf(1) you could just do `expect(movies.models[0].get('release_year')).to.eql(2012);`